update/sync/syncAccountsDBs.go

```diff
@@ -155,7 +155,8 @@ func (st *syncAccountsDBs) syncAccountsOfType(accountType genesis.Type, trieID s
 		return err
 	}
 
-	st.setTries(shardId, accAdapterIdentifier, rootHash, accountsDBSyncer.GetSyncedTries())
+	//TODO(iulian) fix here in this PR
```

Comment:
this whole thing works without this :D maybe from accountsDBSyncer you can take those.

Reply:
will test :+1: